testnet-croeseid-2/1-click-reconfig/README.md

```diff
@@ -0,0 +1,6 @@
+# Upgrade from existing 1-click image
+
+```bash
+$ curl -sSL https://raw.githubusercontent.com/crypto-com/testnets/main/testnet-croeseid-2/1-click-reconfig/reconfig.sh | sudo tee /chain/reconfig.sh >/dev/null
+$ sudo -u crypto /chain/reconfig.sh
+```
```

Comment:
Ask the user to choose `testnet-croeseid-2` as well hahah

Reply:
sure